custom/icds_reports/tests/agg_tests/reports/test_demographics.py

```diff
@@ -165,317 +165,3 @@ def test_data_monthly_adolescent_girls(self):
             "help_text": percent_adolescent_girls_enrolled_help_text()
         }
         self.assertDictEqual(expected, data['records'][2][1])
-
-    def test_data_daily_format(self):
-        data = get_demographics_data(
-            'icds-cas',
-            (2017, 5, 29),
-            {
-                'month': (2017, 5, 1),
-                'prev_month': (2017, 4, 1),
-                'aggregation_level': 1
-            }
-        )
-        self.assertEqual(1, len(data))
-        self.assertEqual(3, len(data['records']))
-        self.assertEqual(2, len(data['records'][0]))
-        self.assertEqual(2, len(data['records'][1]))
-        self.assertEqual(2, len(data['records'][2]))
-
-    def test_data_daily_registered_household(self):
```

Comment:
Why are these test cases deleted?
I assume because we put the function in feature flag?

Reply:
The test case were written for testing daily data cases and since we are pulling from monthly table only now I removed the test cases.

Reply:
> The test case were written for testing daily data cases and since we are pulling from monthly table only now I removed the test cases.(comment from dhiraj pr)

Reply:
i think better to change them accordingly

Reply:
I think its better to keep them  until its under feature flag this will ensure current functionality is not messed up.